### ISSUE TITLE
Sortability of Characters

### ISSUE THREAD

**Issue by Carmcritter** [Ariel-Thomas/adventurers-league-log#126]
Could you please add a feature that either; auto sorts characters alphabetically, or gives the option to sort alphabetically(Other sort options that might be handy would be level, faction, class, etc.)

**Comment by Carmcritter** [comment]
Also... The function of the website so far is amazing, thank you for doing this.

**Comment by Carmcritter** [comment]
I missed that...
Level doesn't seem to be a link, but the rest are an awesome feature.
Thanks muchly

**Comment by Ariel-Thomas** [comment]
Yeah, level is calculated so it's non-trivial to allow sorting. If people really need it for some reason I can put the time in, but until that comes up I won't bother.
